robowflex_library/yaml/test_cob4.yml

```diff
@@ -0,0 +1,41 @@
+name: (noname)
+robot_model_name: cob4
```

Comment:
This should be cob4-8 since that is the name in the srdf

Reply:
Done!